tests/browser_context_test.go

```diff
@@ -0,0 +1,302 @@
+package tests
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBrowserContextAddInitScript(t *testing.T) {
+	t.Parallel()
+
+	t.Run("string_script_on_new_page", func(t *testing.T) {
```

Comment:
Consider putting similar tests in a table test—they have more or less the same body. Let them accept differing parts from the fields. You can add more table tests if things become hard to read.